testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/DemoAppMainActivity.kt

```diff
@@ -20,6 +22,7 @@ import kotlinx.android.synthetic.main.mini_app_main_layout.*
 class DemoAppMainActivity : BaseActivity() {
     private var currentNavController: LiveData<NavController>? = null
     private lateinit var binding: MiniAppMainLayoutBinding
+    val miniAppIdAndViewMap = hashMapOf<String, MiniAppView>()
```

Comment:
if we use same project id and subscription key in both List-I and List-II in demo app, then we are supposed to get multiple miniapps with same `MiniAppId`, therefore we should store the duplicates too. Otherwise, it will work on only one MiniApp which was opened firstly as we are eliminating the duplicate `MiniAppId` using this `HashMap`. And, the other MiniApps with same `MiniAppId` will not work.

I think, `HashMap.put` will be inconvenient to store multiple `MiniAppView`s with duplicate `MiniAppId`. In that case, we can apply other way i.e. `val miniAppIdAndViewMap = mutableListOf<Pair<String, MiniAppView>>()`